collect_app/src/main/java/org/odk/collect/android/formmanagement/FormApiExceptionMapper.java

```diff
@@ -1,23 +1,17 @@
 package org.odk.collect.android.formmanagement;
 
-import android.content.res.Resources;
-
 import org.odk.collect.android.R;
+import org.odk.collect.android.application.Collect;
 import org.odk.collect.android.openrosa.api.FormApiException;
+import org.odk.collect.android.utilities.TranslationHandler;
 
 public class FormApiExceptionMapper {
 
-    private final Resources resources;
-
-    public FormApiExceptionMapper(Resources resources) {
```

Comment:
I think it'd still be nice for this object to take in `Context` as a constructor dependency so it doesn't have to rely on `Collect.getInstance()`

Reply:
Done.